source/use-cases/client-side-field-level-encryption-guide.txt

```diff
@@ -158,12 +160,22 @@ compliant to data privacy regulations with MongoDB.
 Implementation
 --------------
 
-This guide is meant for users new to Client-Side Field Level Encryption (CSFLE).
-It covers the why and how at a high level, and is not meant to produce working
-code in a single file.
+This section explains the following configuration and implementation
+details of CSFLE:
+
+- Software required to run your client and server in your local development
+  environment.
+
+- Creation, setup, and validation of the encryption keys.
```

Comment:
What does "setup" refer to here?

Reply:
I think I meant configuration, and possibly storage. I'll make it more clear.